src/Files.App/Views/Layouts/BaseLayoutPage.cs

```diff
@@ -66,6 +66,7 @@ public abstract class BaseLayoutPage : Page, IBaseLayoutPage, INotifyPropertyCha
 
 		private bool shiftPressed;
 
+		private ListedItem? clickedItem = null;
```

Comment:
Can we merge `clickedItem` to `SelectedItem`? I don't think they have to be separate.

Reply:
> Can we merge `clickedItem` to `SelectedItem`? I don't think they have to be separate.

I haven't quite figured out what `SelectedItem` does, but I'll have a try.